src/main/java/multiversx/MessageSigning.java

```diff
@@ -1,14 +1,11 @@
-package elrond;
+package multiversx;
 
-import org.bouncycastle.crypto.digests.KeccakDigest;
 import org.bouncycastle.crypto.params.Ed25519PrivateKeyParameters;
 import org.bouncycastle.crypto.params.Ed25519PublicKeyParameters;
 import org.bouncycastle.crypto.signers.Ed25519Signer;
 import org.bouncycastle.jcajce.provider.digest.Keccak;
 import org.bouncycastle.util.encoders.Hex;
 
-import java.nio.charset.StandardCharsets;
-
 public class MessageSigning {
     private static final String signerMessagePrefix = "\u0017Elrond Signed Message:\n";
```

Comment:
so this will stay with Elrond?

Reply:
For compatibility, yes. Will decide later